portfolio/src/main/webapp/script.js

```diff
@@ -19,17 +19,28 @@ let a;
 function randomizeImage() {
     const location = [
         {lat:37.422, lng: -122.084},
-        {lat: 39.9514829982124, lng: 32.82864193896349 },
-        {lat:-0.6182972019850904, lng:73.09703007580428 },
-        {lat: 24.470814082528822, lng: 39.617508263244964 },
-        {lat: 36.10039521673435, lng: -112.11208148905413 },
-        {lat: 37.76768251244913, lng: -122.43029199523379 },
-        {lat: 35.06307913148578, lng: 135.74217581612047 },
-        {lat: 41.00549054415733, lng: 28.976631407284994 },
-        {lat: 35.63599072424489, lng:139.63997813974325 },
-        {lat: 6.520748303489362, lng: 3.3560439057592677 },
-        {lat: 35.06307913148578, lng: 135.74217581612047 },
-        {lat: 1.3675398068198992, lng: 103.8737622961487}
+        {lat: 39.9514829982124, lng: 32.82864193896349 , contentString: '<h1> Ankara </h1>' +
```

Comment:
again, maybe for future projects, consider implementing JS and HTML in a way that you don't need to hardcode HTML tags to JS fields

Reply:
one idea (again can keep as an idea for future projects) is to use a Java object, for example in this case would include lat, lng, locationName, locationFact. Then for each type of fields, use a certain HTML formatting style, for example use <h1> for all locationName